test/src/597-graft/main

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
A few more things to test would be
 - a grafted file in a sub directory / nested catalog
 - graft markers for symlinks, directory entries (should perhaps result in a warning)
 - remove a grafted file